strategy-cache-and-update/server.js

```diff
@@ -0,0 +1,20 @@
+var MAX_IMAGES = 50;
+var imageNumber = 0;
+
+module.exports = function(app, route) {
+  app.get(route + 'asset', function(req, res) {
+    serveImage(res, 10000);
+  });
+};
+
+var lastUpdate = -Infinity;
```

Comment:
Because is the translation of `never` but you're right, it's confusing. Let it to be 0. You game an idea for a post. :)